src/lpcjobqueue/cluster.py

```diff
@@ -24,12 +24,11 @@
 
 logger = logging.getLogger(__name__)
 fn = os.path.join(os.path.dirname(__file__), "config.yaml")
-dask.config.ensure_file(source=fn)
```

Comment:
This copies the file to `~/.config/dask/config.yaml` which seems to be for users to optionally override. Since this package is for a specific infrastructure (docker-coffea-dask), we would be better off putting the config in the docker container.